internal/rootcoord/import_manager.go

```diff
@@ -684,9 +685,9 @@ func (m *importManager) setImportTaskStateAndReason(taskID int64, targetState co
 	return nil
 }
 
-func (m *importManager) setCollectionPartitionName(colID, partID int64, task *datapb.ImportTaskInfo) error {
+func (m *importManager) setCollectionPartitionName(dbName string, colID, partID int64, task *datapb.ImportTaskInfo) error {
```

Comment:
No need to add this dbName parameter？

Reply:
there still pass  db name parameter,  because it need to combine db id and collection id to find the key from etcd.